bin/templates/project/app/build.gradle

```diff
@@ -40,7 +40,7 @@ allprojects {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '4.10.3'
+    gradleVersion = '6.0'
```

Comment:
aaaand 6.0.1 is out: https://github.com/gradle/gradle/releases/tag/v6.0.1

Reply:
Yah, I've been passively keeping this PR up to date with the latest version and before the merge I'd suggest we do one last final check.

Reply:
I know, was just a friendly reminder :)

Reply:
Note to myself mainly but before I commit to 6.x, I need to double check things still works fine in android studio, because according to the [docs](https://developer.android.com/studio/releases/gradle-plugin) the latest gradle plugin only supports up to 5.6.4.

Been awhile since I looked at this but in my testing I probably only used the cordova command line.

Reply:
Bumped Gradle to `6.1`
https://github.com/gradle/gradle/releases/tag/v6.1.0